templates/static/accessibility.html

```diff
@@ -0,0 +1,138 @@
+
+  <p>{{ _("Some radio inputs on the relationships page do not conform to GDS guidelines. When selected, three different elements on the page are updated.") }}</p>
+
+  <p>{{ _("On ‘Who lives here?’ there is a ‘Previous’ link on the page that does not conform with GDS guidelines. The text used should read as ‘Back’.") }}</p>
+
+  <p>{{ _("Error messages do not result in a change to the page title to reflect that an error has been made. This does not conform to GDS guidelines.") }}</p>
+
+  <h2>{{ _("How we tested this questionnaire") }}</h2>
+
+  <p>{{ _("Over the last few years we have tested the full questionnaire, including:") }}</p>
+
+    <ul class="list">
+    <li>{{ _("<a href='{url}'>Digital Accessibility Centre (DAC)</a> carrying out a professional, technical audit").format(url = "https://digitalaccessibilitycentre.org/") }}</li>
+    <li>{{ _("user testing with the public on different devices, different locations, in their homes with various assistive technologies including free and paid versions") }}</li>
+    <li>{{ _("testing with charities, such as Scope and RNIB in their labs") }}</li>
+    <li>{{ _("using job centres and libraries for pop-up testing") }}</li>
+    <li>{{ _("covering a wide range of ages, additional needs and disabilities") }}</li>
+    <li>{{ _("making sure the offline journey and non-digital products were considered in the end-to-end journey") }}</li>
+    <li>{{ _("changing designs based on feedback from user testing, such as letter structure and the summary of sections and progress through questionnaire") }}</li>
+  </ul>
+
+  <h3>{{ _("Digital Accessibility Centre testing") }}</h3>
+
+  <p>{{ _("Parts of this questionnaire were last tested on 3 September 2019 by the Digital Accessibility Centre (DAC).") }}</p>
+
+
+  <p>{{ _("<a href='{url}' download>DAC Report - September 2019</a>").format(url = "https://storage.cloud.google.com/census-ci-craftcms/assets/ONS-Census-Survey-2021-DAC-Accessibility-Report-Sept-2019.pdf") }}</p>
+
+  <p>{{ _("You can read the last full accessibility test report from 28 June 2019.") }}</p>
+
+  <p>{{ _("<a href='{url}' download>DAC Report - June 2019</a>").format(url = "https://storage.cloud.google.com/census-ci-craftcms/assets/ONS-Census-Survey-2021-DAC-Accessibility-Report-June-2019.pdf") }}</p>
+
+  <h2>{{ _("What we’re doing to improve accessibility") }}</h2>
+
+  <p>{{ _("We are continuing to work with various charities and local deaf communities to understand their needs for the census.") }}</p>
+
+  <p>{{ _("We are looking at designs to see if it is possible to have British Sign Language on every page. This is challenging as we have been unable to find another service that does this, but we continue to investigate.") }}</p>
+
+  <p>{{ _("We will act on feedback from the 2019 Rehearsal in October.") }}</p>
+
+  <p>{{ _("This statement was prepared on 17 September.") }}</p>
+
+  <p>{{ _("This statement was published on 17 September.") }}</p>
```

Comment:
This could be part of the same sentence as above. Not a new para according to docs